cs1_lab01.Rmd

```diff
@@ -0,0 +1,101 @@
+---
+number: 1
+title: C programming 1
+author:
+course: Computer Science 1
+material: Instruction 1
+---
+
+## 1.
+Create and execute the following program:
+```c++
+#include <stdio.h>          // header for standard input/output
+
+int main()                  // definition of the main function
+{                           // main function starting bracket
+  printf("Hello!\n");
+  return 0;
+}                           // main function ending bracket
+```
+Declare two integer variables (a and b) and print results of: sum, subtraction, multiplication and division.
+The results should be printed in a single line.
+
+## 2.
+Write a program which reads from a keyboard two numbers of double type and prints results of the following operations: multiplication, division, square root, cube root, fourth power, sinus and cosinus.
```

Comment:
the keyboard
Polish: sinus, cosinus -> English: sine, cosine

Reply:
Corrected.